File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraPrinting.Preview;
using DevExpress.XtraReports.Design;
using DevExpress.XtraReports.UI;

namespace WindowsFormsApplication3 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }
        private void simpleButton1_Click(object sender, EventArgs e) {
            ShowDesigner();
        }
        private void ShowDesigner() {
            ReportDesignTool dt = new ReportDesignTool(new XtraReport1());
            dt.DesignForm.DesignMdiController.DesignPanelLoaded += DesignMdiController_DesignPanelLoaded;
            dt.ShowDesigner();
        }
        private void DesignMdiController_DesignPanelLoaded(object sender, DevExpress.XtraReports.UserDesigner.DesignerLoadedEventArgs e) {
            ReportTabControl tabControl = e.DesignerHost.GetService(typeof(ReportTabControl)) as ReportTabControl;
            if(tabControl == null) {
                return;
            }
            tabControl.SelectedTabIndexChanged += tabControl_SelectedTabIndexChanged;
        }
        private void tabControl_SelectedTabIndexChanged(object sender, EventArgs e) {
            ReportTabControl tabControl = sender as ReportTabControl;
            if(tabControl != null && tabControl.PreviewReport != null) {
                DevExpress.XtraBars.Docking.DockPanel paramsPanel = GetParamsPanel(tabControl);
                if(paramsPanel != null) {
                    paramsPanel.Width = 500;
                }
            }
        }
        private DevExpress.XtraBars.Docking.DockPanel GetParamsPanel(ReportTabControl tabControl) {
            return tabControl.PreviewControl.DockManager.Panels.FirstOrDefault(x => x.Text == GetParametersPanelText());
        }
        private System.String GetParametersPanelText() {
            return DevExpress.XtraPrinting.Localization.PreviewLocalizer.GetString(DevExpress.XtraPrinting.Localization.PreviewStringId.ParametersRequest_Caption);
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraPrinting.Preview
Imports DevExpress.XtraReports.Design
Imports DevExpress.XtraReports.UI

Namespace WindowsFormsApplication3
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub
        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
            ShowDesigner()
        End Sub
        Private Sub ShowDesigner()
            Dim dt As New ReportDesignTool(New XtraReport1())
            AddHandler dt.DesignForm.DesignMdiController.DesignPanelLoaded, AddressOf DesignMdiController_DesignPanelLoaded
            dt.ShowDesigner()
        End Sub
        Private Sub DesignMdiController_DesignPanelLoaded(ByVal sender As Object, ByVal e As DevExpress.XtraReports.UserDesigner.DesignerLoadedEventArgs)
            Dim tabControl As ReportTabControl = TryCast(e.DesignerHost.GetService(GetType(ReportTabControl)), ReportTabControl)
            If tabControl Is Nothing Then
                Return
            End If
            AddHandler tabControl.SelectedTabIndexChanged, AddressOf tabControl_SelectedTabIndexChanged
        End Sub
        Private Sub tabControl_SelectedTabIndexChanged(ByVal sender As Object, ByVal e As EventArgs)
            Dim tabControl As ReportTabControl = TryCast(sender, ReportTabControl)
            If tabControl IsNot Nothing AndAlso tabControl.PreviewReport IsNot Nothing Then
                Dim paramsPanel As DevExpress.XtraBars.Docking.DockPanel = GetParamsPanel(tabControl)
                If paramsPanel IsNot Nothing Then
                    paramsPanel.Width = 500
                End If
            End If
        End Sub
        Private Function GetParamsPanel(ByVal tabControl As ReportTabControl) As DevExpress.XtraBars.Docking.DockPanel
            Return tabControl.PreviewControl.DockManager.Panels.FirstOrDefault(Function(x) x.Text = GetParametersPanelText())
        End Function
        Private Function GetParametersPanelText() As System.String
            Return DevExpress.XtraPrinting.Localization.PreviewLocalizer.GetString(DevExpress.XtraPrinting.Localization.PreviewStringId.ParametersRequest_Caption)
        End Function
    End Class
End Namespace
